lib/components/viewers/stop-live-table.js

```diff
@@ -0,0 +1,157 @@
+import moment from 'moment'
+import 'moment-timezone'
+import coreUtils from '@opentripplanner/core-utils'
+import React, { Component } from 'react'
+
+import Icon from '../narrative/icon'
+import PatternRow from './pattern-row'
+import {
+  getRouteIdForPattern,
+  getStopTimesByPattern,
+  routeIsValid
+} from '../../util/viewer'
+
+const { getUserTimezone } = coreUtils.time
+
+const defaultState = {
+  spin: false,
+  timer: null
+}
+
+class StopLiveTable extends Component {
```

Comment:
Maybe a better component name would be `LiveStopArrivalsByPattern`?

Reply:
Renamed in. https://github.com/opentripplanner/otp-react-redux/pull/292/commits/4bd58959771c9acc35dd8eeea1bb5e4ace13c818.